input/definitions/Measure/StructureDefinition-cqf-improvementNotationGuidance.json

```diff
@@ -0,0 +1,87 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "cqf-improvementNotationGuidance",
```

Comment:
Add the extensions to indicate wg ownership (cqi), fmm status (1), and standards-status (trial-use) See some of the other extensions for examples